spacy/language.py

```diff
@@ -1173,6 +1168,14 @@ def update(
                     examples,
                 ):
                     eg.predicted = doc
+        for name, proc in self.pipeline:
```

Comment:
Maybe add a comment here to explain why it's a separate loop?

Reply:
Ah, yeah, good idea, helps avoiding that the loops get fused again.